Android/Narla Komal Kalyan/Advanced/NewsFeed/app/src/main/java/com/example/newsfeed/FeedDetails.java

```diff
@@ -0,0 +1,132 @@
+package com.example.newsfeed;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.resource.drawable.DrawableTransitionOptions;
+import com.bumptech.glide.request.RequestOptions;
+import com.google.android.material.appbar.AppBarLayout;
+import com.google.android.material.appbar.CollapsingToolbarLayout;
+
+public class FeedDetails extends AppCompatActivity implements AppBarLayout.OnOffsetChangedListener{
+
+    private ImageView imageView;
+    private TextView appbar_title, appbar_subtitle, date, time, title;
+    private boolean isHideToolbarView = false;
+    private FrameLayout date_behavior;
+    private LinearLayout titleAppbar;
+    private AppBarLayout appBarLayout;
+    private Toolbar toolbar;
+    private String mUrl, mImg, mTitle, mDate, mSource, mAuthor;
```

Comment:
Most of the fields can be converted to local variables.

Reply:
Done